### PR TITLE
feat(webgpu-rendering-context): import webgpu renderer

### PR DESCRIPTION
**Pull request by esterTion**
This PR finishes #74

### CODE REVIEW

**Review by Lutra-Fs — CHANGES_REQUESTED** [review]
node_modules\three\examples\jsm\capabilities\WebGPU.js
or how do we add the three addons? needs to specify these instructions first.

**Comment by esterTion** [comment]
they are already in the three package, and import will work with current configuration. 
the only thing required is typescript declaration of the type, since these modules are not released yet

File: src/components/Simulation.tsx [hunk 1]
```diff
@@ -59,6 +63,15 @@ function DiffusionPlane(
   props: ThreeElements['mesh'] & Renderable,
 ): React.ReactElement {
   // INITIALISATION
+  
+  // WebGPU capability test
+  if ( WebGPU.isAvailable() === true ) {
```
Comment (Lutra-Fs) [inline src/components/Simulation.tsx:68]:
```
CFlowSim\src\components\Simulation.tsx
  68:8   error  Unsafe call of an `any` typed value  @typescript-eslint/no-unsafe-call
  69:11  error  Unsafe assignment of an `any` value  @typescript-eslint/no-unsafe-assignment
```
line returns these errors in `npm run lint`

Comment (Lutra-Fs) [reply src/components/Simulation.tsx:68]: Also pls make sure that you have configured `prettier` as the formatter.

**Review by Lutra-Fs — CHANGES_REQUESTED** [review]

**Comment by Lutra-Fs** [comment]
> they are already in the three package, and import will work with current configuration. the only thing required is typescript declaration of the type, since these modules are not released yet

Understood. pls resolve format and linting error to get it merged

**Review by Lutra-Fs — APPROVED** [review]
LGTM

**Review by peipacut — APPROVED** [review]

**Pull request merged**